crates/matrix-sdk-crypto/src/verification/machine.rs

```diff
@@ -650,7 +649,6 @@ mod tests {
         assert!(!alice.timed_out());
         assert!(alice_machine.verifications.outgoing_requests().is_empty());
 
-        // This line panics on macOS, so we're disabled for now.
```

Comment:
This always compiled, but it produces a runtime panic, is this not the case anymore?

Reply:
Test passes without issue, so I'd certainly say so :)
```
        PASS [   0.014s] matrix-sdk-crypto verification::machine::tests::timing_out
```